src/components/PreviewFrame.jsx

```diff
@@ -190,8 +186,12 @@ class PreviewFrame extends React.Component {
     this._channel = Channel.build({
       window: frame.contentWindow,
       origin: '*',
```

Comment:
It occurs to me, I think we can set this to `window.location.origin` since the frame now has a real origin?

Reply:
Hmm never mind I just poked around with this and it does not work straightforwardly…